.github/workflows/release.yml

```
@@ -24,10 +24,13 @@ jobs:
           wget -O- https://github.com/intuit/auto/releases/download/v10.16.1/auto-linux.gz | gunzip > ~/auto
           chmod a+x ~/auto
 
-      - name: Check whether a release is due
+      - name: Query 'auto' on type of the release
         id: auto-version
         run: |
-          version="$(~/auto version)"
+          # to be able to debug if something goes wrong
+          set -o pipefail
+          auto version -v | tee /tmp/auto-version
```

```suggestion
          auto version -vv | tee /tmp/auto-version
```

Use the same verbosity as in https://github.com/dandi/dandi-cli/blob/aa55b8914c8ff2216780892e5ff1796d350fd0a4/.github/workflows/release.yml#L34